docs/users/allowed-code.rst

```diff
@@ -0,0 +1,87 @@
+=============
+Code Allowed in WordPress/Largo Posts and Pages
```

Comment:
hm, not sure we need to include this entire list here, i'd rather just link to it on wp.org. Particularly since we wouldn't actually encourage people to use the majority of these tags in a post/page, and also because if wp.org changes this list I'd be willing to bet we won't catch it and then this will be out of date, etc.